twitter_analytics_client.rb

```diff
@@ -34,11 +34,11 @@ def get_analytics_data
   private
 
   def start_date
-    @start_date ||= ( Time.now.beginning_of_month.since(9.hours)).to_i.to_s + "000"
+    @start_date ||= ( Time.now.utc.beginning_of_month).to_i.to_s + "000"
   end
 
   def end_date
-    @end_date ||= (Time.now.since(9.hours)).to_i.to_s + "999"
+    @end_date ||= (Time.now.utc).to_i.to_s + "999"
```

Comment:
I solved the problem that date_range is not equal to what I intended.
I can get data from the beginning of the month to now.